tox.ini

```diff
@@ -29,6 +29,15 @@ deps =
 
 commands = mypy web_poet tests
 
+[testenv:types]
+deps =
+    {[testenv]deps}
+    {[testenv:mypy]deps}
+    https://github.com/kmike/pytest-mypy-testing/archive/refs/heads/async-support.zip
```

Comment:
it's needed because of https://github.com/davidfritzsche/pytest-mypy-testing/pull/31